sysutils/istioctl/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                istioctl
+
+github.setup        istio istio 1.0.2
+categories          sysutils
+platforms           darwin
+supported_archs     x86_64
+license             Apache-2
+
+maintainers         {vmware.com:nnikolay} openmaintainer
+
+description         Istio configuration utility
+long_description    Create, list, modify, and delete configuration \
+                    resources in the Istio system.
```

Comment:
These descriptions should probably be better. (In particular, it's rather difficult to know what this port is for from reading them. What's Istio, for example?)

Reply:
Done, thanks.